Makefile

```diff
@@ -31,6 +31,13 @@ all build:
 	hack/build-go.sh $(WHAT)
 .PHONY: all build
 
+# Build Code with E2E tests artifact.
+# Example:
+# make e2e
+e2e:
+	hack/build-go.sh test/e2e/e2e.test
```

Comment:
Hmmmmm https://github.com/openshift/origin/blob/master/test/extended/util/test.go Seems to be wrapping the existing tests with a customized runner that replaces e2e.go, by imementing the same functionality.